astropy/wcs/tests/test_pickle.py

```diff
@@ -86,7 +88,14 @@ def test_wcs():
     header = get_pkg_data_contents(
         os.path.join("data", "outside_sky.hdr"), encoding='binary')
 
-    wcs1 = wcs.WCS(header)
+    if sys.platform.startswith('win'):
```

Comment:
@nden , is this expected?

Reply:
If this ends up not being desired behavior, I'd suggest changing to a ``pytest.mark.filterwarnings``, and same below.